org.uqbar.project.wollok.tests/src/org/uqbar/project/wollok/tests/typesystem/xpect/console.wpgm.xt

```diff
@@ -13,7 +13,7 @@ program p {
 	// XPECT type at console --> console 
  	console
 
-	// XPECT methodType at println --> (String) => Void 
+	// XPECT methodType at println --> (Any) => Void 
```

Comment:
@fdodino please check this: method println in console wko is annotated as Any, not String.